lib/graphql/iterableModel.ts

```diff
@@ -31,4 +34,17 @@ export abstract class IterableModel<T> extends Model<T> {
       }
     };
   }
+
+  [Symbol.asyncIterator]() {
```

Comment:
As mentioned in the PR description: 

Should we keep this iterator on the base `iterableModel`, even if it is not usable for transfers? should we deprecate it? should we completely remove it (I am almost sure the webapp doesn't use it yet)? Removing it would obviously simplify the logic a little bit

Reply:
Let’s remove it there, will not be usable for transfers anyway as you already said.

Reply:
All right, I agree, removed here: 3886585a94da3caabd3a84dab3c7f7ed89c52b75